### DISCUSSION TITLE
Better color extraction?

### DISCUSSION THREAD

**Discussion by hanubeki** [Mahmud0808/ColorBlendr#89] [Ideas]
Currently ColorBlendr seems to be using androidx Palette that is limited to vibrant (dark/light) and muted (dark/light) colors but quantize from monet library could give more colors.

https://github.com/Mahmud0808/ColorBlendr/tree/master/app/src/main/java/com/drdisagree/colorblendr/utils/monet/quantize

For live wallpapers, as of Android 8.1, live wallpapers can report wallpaper colors that could be used for theming.

https://developer.android.com/reference/android/app/WallpaperColors

LWP+ app is an example for utilization of WallpaperColors.

https://play.google.com/store/apps/details?id=com.lb.lwp_plus

**Comment by Mahmud0808** [comment]
Is there any documentation available for the quantize of monet library?

**Reply by hanubeki** [reply]
https://github.com/material-foundation/material-color-utilities/blob/main/extract_colors.md

I think this is how to extract colors using quantize and score.

**Reply by Mahmud0808** [reply]
https://github.com/Mahmud0808/ColorBlendr/issues/91
